system_setup/server/controllers/configure.py

```diff
@@ -40,89 +41,189 @@ def __init__(self, app):
     def start(self):
         self.install_task = self.app.aio_loop.create_task(self.configure())
 
-    def _update_locale_gen(self, localeGenPath, lang):
-        """ Uncomments the line in locale.gen file where lang is found,
-            if found commented. A fully qualified language is expected,
-            since that would have passed thru the Locale
-            controller validation. e.g. en_UK.UTF-8. """
+    def __locale_gen_cmd(self) -> Tuple[str, bool]:
+        """ Returns the locale-gen command path if not in dry-run.
+            Otherwise, copies the locale-gen script, altering the localedef
+            command line to output into a specified directory.
+            Additionally, indicates success by returning True
+            as the second element of the tuple. """
+
+        cmd = "usr/sbin/locale-gen"
```

Comment:
TBH, I’m not a big fan of it, but I understand why you did this: the fact dry-run can be executed outside of tests only makes mangling the `PATH` env variable more annoying (and you need to ensure you have the local copy somewhere, also…), so I think in the python world, this a good approach (I will show you how this is done generally in Golang, but I dno’t think this should be applied here).

So, it’s well done, I don’t have other counter-proposal, let’s go with this.